src/super_gradients/training/datasets/dataset_interfaces/dataset_interface.py

```diff
@@ -732,10 +747,13 @@ def __init__(self, dataset_params=None, cache_labels=False, cache_images=False):
                                                          sample_loading_method=train_sample_method,
                                                          cache_labels=cache_labels,
                                                          cache_images=cache_images,
-                                                         augment=True)
+                                                         augment=True,
+                                                         class_inclusion_list=class_inclusion_list)
                 train_sets.append(sub_trainset)
```

Comment:
We are training on both train **and val**?

Reply:
Yes that's the common convention for VOC. Train on 2007 2012 trainval, and validate on 2007 test.